v3/doc/modules.md

```diff
@@ -0,0 +1,114 @@
+It is possible to have a single top level eslint config, but then you can't lint just a single package at a time (as far as I can tell by reading that issue)
+
+## Postcss
+Seems like this is mainly used for the autoprefixer plugin which adds vendor specific prefixes based on the caniuse data.
+
+It is loaded by webpack using the postcss-loader. This wepback config doesn't explicitly declare the location of the config file. I'd guess that it looks for it relative to the webpack config. So it might work to have a common config in the root. However one recommendation is to have a common config in a "config" package and then import that into separate "empty" configs in each code package: https://shubhamverma.me/blog/setup-tailwind-css-with-turborepo
+Linked from here: https://github.com/tailwindlabs/tailwindcss/discussions/9890
+
+# Peer dependency comparison
+NPM from v3-v6 did not automatically install peer dependencies.
+Since v7, it started installing peer dependencies automatically.
+NPM also became strict about conflicting peer dependencies with v7, so if there are conflicts it will cause an error and not finish the dependency installation.
+
+Yarn v2, v3, and v4 are also supposed to automatically install peer dependencies. However when combined with the strict dependency checking of the PnP install mode, these implicit dependencies cannot not be imported or required. So it is necessary add these peer dependencies at least to the top level package.json so they can be used.
+
+For completeness there are also optional peer dependencies which are supported by both yarn and npm. These will show up in yarn's `yarn explain peer-requirements`, but they shouldn't cause errors if they are not installed. If used properly they shouldn't cause errors if they aren't added to the any of the package.json files.
+
+# Yarn migration:
+- removed node_modules
+- ran `yarn import` to create yarn.lock from package-lock.json
+- ran `yarn set version stable` embed the version 4 of yarn in the project this way it will be used regardless of what version users have globally
+- switched to using yarn's built in patch support:
+  - removed patch package from dependencies and scripts
+  - for each patch in `/patches`:
+    - `yarn patch [package name]`
+    - `cd [temporary folder created by yarn]`
+    - `patch -p4 < [path to patch file from patch package in /patches]` Note: sometimes -p3 is needed. This depends on the number of path segments in the patch file you want to ignore when applying a patch. For organization packages like `@nedb/binary-search-tree`, `-p4` is needed. For top level packages like `react-data-grid`, `-p3` is needed.
+    - `cd -`
+    - `yarn patch-commit -s [temporary folder created by yarn]`
+  - updated references to "patch package" in the source code
```

Comment:
One question that could be answered here is what the patching process looks like moving forward. It used to be that you'd edit the files in the `node_modules` folder and then `patch_package` would compare the version in the `mode_modules` folder with the version on npm. What does the patch-development process look like using yarn's patch mechanism?